students/jemielity_kamil/lesson_02_flow_control/ladder.py

```diff
@@ -0,0 +1,8 @@
+
+number = int(input('Integer: '))
+
+for row in range(1, number+1):
+    for i in range(1, row+1):
+        print(i, end='')
+
+    print(sep='')
```

Comment:
Isn't this print just for an empty line? If yes then it enough to print empty string print('')